cmds/core/tftp/client.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2012-2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"io"
+
+	"pack.ag/tftp"
+)
+
+type Response interface {
+	Read([]byte) (int, error)
```

Comment:
here and everywhere, public vars / structs / function, need a comment in form of `<> does abc ...` e.g. 

```
// Response ....
type Resobse interface {
   ..
}
```

Reply:
done